src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -50,7 +50,6 @@ class ENAButton: DynamicTypeButton {
 	@IBInspectable var isInverted: Bool = false { didSet { applyStyle() } }
 	@IBInspectable var isLoading: Bool = false { didSet { applyStyle() } }
 	@IBInspectable var hasBorder: Bool = false { didSet { applyStyle() } }
-	// swiftlint:disable redundant_optional_initialization
```

Comment:
You will have to re-add this to make the linter happy again :D

Reply:
Strange, a warning occurred that this does not trigger something. But now the linter wants it back....I rolled back my change.